get_facts on iosxr fails

```
    def get_facts(self):
    
        facts = {
            'vendor': u'Cisco',
            'os_version': u'',
            'hostname': u'',
            'uptime': -1,
            'serial_number': u'',
            'fqdn': u'',
            'model': u'',
            'interface_list': []
        }
    
        facts_rpc_request = '<Get><Operational><SystemTime/><PlatformInventory/>\
            </Operational></Get>'
    
        facts_rpc_reply = ETREE.fromstring(self.device.make_rpc_call(facts_rpc_request))
        system_time_xpath = './/SystemTime/Uptime'
        platform_attr_xpath = './/RackTable/Rack/Attributes/BasicInfo'
        system_time_tree = facts_rpc_reply.xpath(system_time_xpath)[0]
>       platform_attr_tree = facts_rpc_reply.xpath(platform_attr_xpath)[0]
E       IndexError: list index out of range

../napalm/napalm/iosxr/iosxr.py:186: IndexError
```

Using the develop branch as of today.

Running IOS-XRv.

```
Sat Nov 18 00:40:17.546 UTC
Cisco IOS XR Software, Version 5.3.1[Default]
Copyright (c) 2015 by Cisco Systems, Inc.
ROM: GRUB, Version 1.99(0), DEV RELEASE
pynet-iosxr1 uptime is 39 weeks, 56 minutes
System image file is "bootflash:disk0/xrvr-os-mbi-5.3.1/mbixrvr-rp.vm"
cisco IOS XRv Series (Intel 686 F6M13S3) processor with 4193904K bytes of memory.
Intel 686 F6M13S3 processor at 2673MHz, Revision 2.174
IOS XRv Chassis
3 GigabitEthernet
1 Management Ethernet
```

@mirceaulinic I just tested this and still fails as of current `develop` branch

```
test_napalm_iosxr.py::test_facts FAILED

================================================================== FAILURES ==================================================================
_________________________________________________________________ test_facts _________________________________________________________________

napalm_connect = <napalm.iosxr.iosxr.IOSXRDriver object at 0xb6005b6c>

    def test_facts(napalm_connect):
>       napalm_facts = napalm_connect.get_facts()

test_napalm_iosxr.py:2: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <napalm.iosxr.iosxr.IOSXRDriver object at 0xb6005b6c>

    def get_facts(self):
    
        facts = {
            'vendor': u'Cisco',
            'os_version': u'',
            'hostname': u'',
            'uptime': -1,
            'serial_number': u'',
            'fqdn': u'',
            'model': u'',
            'interface_list': []
        }
    
        facts_rpc_request = '<Get><Operational><SystemTime/><PlatformInventory/>\
            </Operational></Get>'
    
        facts_rpc_reply = ETREE.fromstring(self.device.make_rpc_call(facts_rpc_request))
        system_time_xpath = './/SystemTime/Uptime'
        platform_attr_xpath = './/RackTable/Rack/Attributes/BasicInfo'
        system_time_tree = facts_rpc_reply.xpath(system_time_xpath)[0]
>       platform_attr_tree = facts_rpc_reply.xpath(platform_attr_xpath)[0]
E       IndexError: list index out of range

../../napalm/napalm/iosxr/iosxr.py:186: IndexError
```

--- a/napalm/iosxr/iosxr.py
+++ b/napalm/iosxr/iosxr.py
@@ -183,7 +183,10 @@ class IOSXRDriver(NetworkDriver):
         system_time_xpath = './/SystemTime/Uptime'
         platform_attr_xpath = './/RackTable/Rack/Attributes/BasicInfo'
         system_time_tree = facts_rpc_reply.xpath(system_time_xpath)[0]
-        platform_attr_tree = facts_rpc_reply.xpath(platform_attr_xpath)[0]
+        try:
+            platform_attr_tree = facts_rpc_reply.xpath(platform_attr_xpath)[0]
+        except IndexError:
+            platform_attr_tree = facts_rpc_reply.xpath(platform_attr_xpath)
 
         hostname = napalm.base.helpers.convert(
             text_type, napalm.base.helpers.find_txt(system_time_tree, 'Hostname'))

